src/test/kotlin/org/rust/ide/annotator/RsStdlibErrorAnnotatorTest.kt

```diff
@@ -24,4 +24,9 @@ class RsStdlibErrorAnnotatorTest : RsAnnotatorTestBase(RsErrorAnnotator::class.j
 
         <error descr="Can't find crate for `litarvan` [E0463]">extern crate litarvan;</error>
     """)
+
+    fun `test missing feature E0132`() = checkErrors("""
```

Comment:
Hmm, why is this test not in `RsErrorAnnotatorTest.kt`? `RsErrorAnnotatorTest` already has a few tests for E0658.

Reply:
Without stdlib `START.availbility` returns unknown. That means that without stdlib the tests would pass too. I could've left `#[feature]` out in the non stdlib tests but I added them just in case

Reply:
> Without stdlib START.availbility returns unknown.

Just add `@MockRustcVersion` annotation:
```kotlin
@MockRustcVersion("1.0.0")
fun `test start feature E0658 1`() = checkErrors("""
    #[<error descr="#[start] function is experimental [E0658]">start</error>]
    fn start(_argc: isize, _argv: *const *const u8) -> isize { 0 }
""")

@MockRustcVersion("1.0.0")
fun `test start feature E0658 2`() = checkErrors("""
    #![feature(start)]
    #[<error descr="#[start] function is experimental [E0658]">start</error>]
    fn start(_argc: isize, _argv: *const *const u8) -> isize { 0 }
""")

@MockRustcVersion("1.0.0-nightly")
fun `test start feature E0658 3`() = checkErrors("""
    #![feature(start)]
    #[start]
    fn start(_argc: isize, _argv: *const *const u8) -> isize { 0 }
""")
```